src/routes/+page.svelte

```diff
@@ -27,7 +27,8 @@
 	<GeocodingControl
 		{apiKey}
 		on:select={(e) => {
-			map.flyTo({
+			if (!e.detail?.center) return;
```

Comment:
:+1: clever detail, would have taken me aaages to come up with that. Magic :sparkles: